api/clients/disperser_client.go

```diff
@@ -38,7 +46,7 @@ func NewConfig(hostname, port string, timeout time.Duration, useSecureGrpcFlag b
 	}
 }
 
-type DisperserClient interface {
+type IDisperserClient interface {
```

Comment:
I don't think we use this convention, this looks solidity

 what do you think @ian-shim

Reply:
I know, I don't like it either, but we already use this for eigenda_client, so I just followed the convention there.
Was forced to make the `disperserClient` struct public for the tests, so had to find another name for the interface. More golangy might be `DisperserClienter`. Honestly I think defining the interface here is code smell and should eventually all be refactored. Interfaces should be defined case-by-case at the call sites.

Reply:
Ok, saw it. we used IEigenDAClient.

Looks fine to me

Reply:
> Was forced to make the disperserClient struct public for the tests

Don't see any place that uses it. We couldn't use the constructor in the test?

Reply:
Ah, you're right. Originally was trying to put the tests in the same package so I can build the struct manually injecting the connection to test something. But then realized I can just add the config parameter.

Reply:
So hmmm... could revert to the old names. But somehow I feel like our clients should follow the same convention.

Reply:
Reverted: [dc98581](https://github.com/Layr-Labs/eigenda/pull/849/commits/dc985811c0bdc2d47e4e046c67b4756294258505)
Also added TODO to change eigenda-client to use same naming convention as disperser-client: [57d716f](https://github.com/Layr-Labs/eigenda/pull/849/commits/57d716fec20db630599a969ce1b4b7ecbe61242c)